ios/Classes/NativeBridge/Handlers/ChangePinHandler.swift

```diff
@@ -1,50 +1,43 @@
 import OneginiSDKiOS
 import Flutter
 
-class ChangePinHandler: NSObject {
-    var changePinCompletion: ((Result<Void, FlutterError>) -> Void)?
-    private let loginHandler: LoginHandler
-    private let registrationHandler: RegistrationHandler
-
-    init(loginHandler: LoginHandler, registrationHandler: RegistrationHandler) {
```

Comment:
Why you remove that? Don't you test this injecting mock objects? Even if you want to use `BridgeConnector.shared` you can have
`init(loginHandler: LoginHandler = BridgeConnector.shared.toLoginHandler, registrationHandler: RegistrationHandler = BridgeConnector.shared.toRegistrationHandler) {`
BTW why BridgeConnector.shared? can be nil?